nni/compression/pytorch/speedup/compressor.py

```diff
@@ -441,12 +450,15 @@ def forward(self, x):
             super_module, leaf_module = get_module_by_name(
                 self.bound_model, g_node.name)
             m_type = g_node.op_type
-            if not m_type in replace_module:
+            if not m_type in replace_module and (self.customized_replace_func is not None and m_type not in self.customized_replace_func):
```

Comment:
suggest to set `self.customized_replace_func` to {} if it is None, then no need to check `self.customized_replace_func` any more.

Reply:
Using a mutable default parameter is dangerous. FYI, https://betterprogramming.pub/mutable-default-arguments-in-python-643ae2583e00